msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.HotRestart.targets

```diff
@@ -86,7 +87,34 @@
 		</Touch>
 	</Target>
 
-	<Target Name="_CollectHotRestartFrameworks" >
+	<!-- Also include frameworks from packages -->
+	<Target Name="_CollectHotRestartFrameworksFromPackages"
```

Comment:
Nothing depends on this target, so as far as I can tell it's never called?

Reply:
Oops - edit fail.  Fixed.